.github/workflows/deploy-test-pypi.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish to TestPyPI
+
+on:
+  push:
+    branches:
+      - 'release/*.*.*'
+
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
+
+jobs:
+  build-and-test-sdist-and-wheels:
+    uses: ./.github/workflows/publish-packages.yml
+    with:
+      upload_to_test: true
+    secrets:
+      TEST_PYPI_API_TOKEN: ${{ secrets.TEST_PYPI_API_TOKEN }}
```

Comment:
I don't see this secret here: https://github.com/glotzerlab/signac-flow/settings/secrets/actions